Xwt.WPF/Xwt.WPFBackend/WindowsSpinButton.xaml.cs

```diff
@@ -458,9 +459,19 @@ private void textBox_MouseWheel(object sender, MouseWheelEventArgs e)
 		#endregion
 
 		#region Accessibility
-		protected override AutomationPeer OnCreateAutomationPeer ()
+		class SpinButtonTextBox : TextBox
 		{
-			return new WindowsSpinButtonAutomationPeer (this);
+			WindowsSpinButton spinButton;
+
+			public SpinButtonTextBox (WindowsSpinButton spinButton)
+			{
+				this.spinButton = spinButton;
+			}
+
+			protected override AutomationPeer OnCreateAutomationPeer ()
```

Comment:
This is not correct, the automation peer needs to be on the overall user control not the inner textbox. It should then not advertise any children element. This is consistent with the rest of the platform and what I was told by the accessibility people.

Reply:
It just didn't work before, when the parent UserControl returned the peer, as it's the TextBox that has the focus and is thus what's queried by e.g. the Narrator for the peer. The spinner control is "mostly" a textbox, so there's some logic in doing this, though we may want to change the parent control to be "pane" instead of "custom".

Anyway, let me ask at office hours today about this. There might be a better way to handle it.

Reply:
I ended up changing changed this slightly. Now the outer spinner UserControl and the TextBox both return the same automation peer, which is associated with the outer spinner. Again, with the original code the Narrator skipped over the spinner controls, not apparently seeing them as accessible when the TextBox inside had the focus. But this changed version seems to work well.

And FYI - I did attend office hours. Guy Barker is the person who wrote much of the Windows automation API. He wasn't there when I attended, but I got his name. I didn't end up reaching out to him for this, but we may reach out in the future.